packages/cozy-client/src/manifest.js

```diff
@@ -0,0 +1,85 @@
+const AUTHORIZED_CATEGORIES = [
```

Comment:
categories of what ? I do not see that self-explainatory. Maybe APP_CATEGORY (and same thing for the function below).

+ I would externalize the list of categories in a json or in its own JS files